tests/foreman/api/test_contentview.py

```diff
@@ -324,11 +324,9 @@ def test_positive_promote_1(self):
                 organization=self.org.id
             ).create_json()['id']
             response = cvv.promote(lc_env_id)
-            self.assertEqual(
-                response['result'],
-                'success',
-                response['humanized']['errors']
-            )
+            humanized_errors = response['humanized']['errors']
+            _check_bz_1186432(humanized_errors)
```

Comment:
This function will raise `unittest.SkipTest` if BZ 1186432 is open and `humanized_errors` contains a message like this:

> ERF12-4115 [ProxyAPI::ProxyException]: Unable to get classes from Puppet for
> KT_FLroGDT3UX_ilMwJQdmVG_xRCEOKGtxC_10 ([RestClient::NotAcceptable]: 406 Not
> Acceptable) for proxy https://<snip>:9090/puppet